### PR TITLE
Fizzbuzz/python/andrew

### PR DESCRIPTION
**Pull request by corrosivecamel**
Adding my name to index.html and adding FizzBuzz directory.

### CODE REVIEW

**Comment by corrosivecamel** [comment]
Thats what it does

**Review by darshandiamond — COMMENTED** [review]
I think the program works fine, but the shell script does ask for a number instead of just typing ./run.sh 17 and getting the output.   I don't know if that was required, but otherwise if looked good to me.

**Comment by Skeegan123** [comment]
> I think the program works fine, but the shell script does ask for a number instead of just typing ./run.sh 17 and getting the output.   I don't know if that was required, but otherwise if looked good to me. 

This is what I meant, also not 100% sure this is required but it seems like this is what is preferred

**Comment by corrosivecamel** [comment]
Ah gotcha. I didn't understand before, but I see what you're saying now.

**Comment by cameronwilcox** [comment]
+1 script + code does appear to work, though I would definitely recommend the changes discussed above.

File: fizzbuzz/index.html
Comment (Tech-Wizz) [inline fizzbuzz/index.html:1]: This link you added does not go anywhere and you are missing the instructions in HTML format. I believe this is where this is supposed to go? Also on top of this, you have your code in a fizzbuzz folder in a fizzbuzz folder. I believe the format is like how you linked to your missing webpage.

File: fizzbuzz/fizzbuzz/run.sh
Comment (Tech-Wizz) [inline fizzbuzz/fizzbuzz/run.sh:1]: I agree with Keegan. The format is supposed to work from git bash so we can't really select a file we just run it in a terminal. It still works so the fizzbuzz is complete

**Review by Tech-Wizz — CHANGES_REQUESTED** [review]
If you can review the small issues you are golden

**Pull request closed without merging**